### DISCUSSION TITLE
v0.1.0-alpha.8

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#61] [Releases]
## What's Changed
* Adhoc speedtest button on dashboard page by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/55
* Moved latest test widgets to the dashboard by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/57
* Replace config.yml with settings UI by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/58

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.1.0-alpha7...v0.1.0-alpha.8

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.1.0-alpha.8'>v0.1.0-alpha.8</a>.</em>

**Comment by alexdelprete** [comment]
Great. I deleted config.yml.

Queued manually (through the new dashboard button) the first test:

![image](https://user-images.githubusercontent.com/7027842/197215732-8be46fbd-e4ae-4398-b115-c6e6130e484f.png)

Two things:
- For the cron expression link suggestion: I'd point it to `https://crontab.guru`, it's the best and simplest.
- In setting speedtest: I used the server ID, but I was not sure, maybe it's better specifying it's the ID and not the hostname.

![image](https://user-images.githubusercontent.com/7027842/197216290-46b8c7b1-2cae-4d46-bec6-36ef690ea4bb.png)

**Comment by alexdelprete** [comment]
I confirm the scheduler is not working, but there's the open issue about it.
